packages/node/src/generators/init/schema.d.ts

```diff
@@ -2,4 +2,5 @@ export interface Schema {
   unitTestRunner?: 'jest' | 'none';
   skipFormat?: boolean;
   js?: boolean;
+  tsConfigName?: string;
```

Comment:
Why add this to the schema in some places but then others you assign it separately?